src/util-napatech.h

```diff
@@ -14,43 +14,51 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
  * 02110-1301, USA.
  */
-
 /**
  * \file
  *
  * \author Phil Young <py@napatech.com>
  *
  */
-
 #ifndef __UTIL_NAPATECH_H__
 #define __UTIL_NAPATECH_H__
 
 #ifdef HAVE_NAPATECH
 #include <nt.h>
 
-typedef struct NapatechPacketVars_ {
+typedef struct NapatechPacketVars_
+{
```

Comment:
nit: suggest consistent placement of `{` on same line with `typedef`

Reply:
Fixed.